### PR TITLE
Update coronavirus.html

### PR DESCRIPTION
**Pull request by hazerb**
Added links to home , worldStats and coronavirusCountryLive pages and made some font changes
Links for author search and profile are deleted

### CODE REVIEW

**Comment by UkcaGreen** [comment]
Style changes seems okay however href links are not working that way in flask. You can check other frontend pages for usage of {{ url_for() }} function.

File: practice-app/templates/coronavirus.html [hunk 1]
```diff
@@ -105,19 +105,13 @@
...
-                    </li>
-                    <li>
-                      <a href="#"
-                        ><i class="fa fa-user" aria-hidden="true"></i
-                      ></a>
-                    </li>
+                     <li><a href="coronavirusCountryLive">Search Country</a></li>
+                  </ul>
+		  <ul class="navbar_user">
+                     <li><a href="worldStats">Worldwide Overview</a></li>
```
Comment (UkcaGreen) [inline practice-app/templates/coronavirus.html:114]: Same applies here.

File: practice-app/templates/coronavirus.html [hunk 2]
```diff
@@ -105,19 +105,13 @@
                 </div>
                 <nav class="navbar">
                   <ul class="navbar_menu">
-                    <li><a href="#">home</a></li>
+                    <li><a href="/">home</a></li>
```
Comment (UkcaGreen) [inline practice-app/templates/coronavirus.html:108]: This href link is not working that way in flask. You can check other frontend pages for usage of {{ url_for() }} function.

File: practice-app/templates/coronavirus.html (hunk 1 above)
Comment (hazerb) [reply practice-app/templates/coronavirus.html:114]: i checked  usage of {{ url_for() }} function, but my code seems fine too. It uses relative path linking, thank you for your review.

File: practice-app/templates/coronavirus.html (hunk 2 above)
Comment (hazerb) [reply practice-app/templates/coronavirus.html:108]: i checked  usage of {{ url_for() }} function, but my code seems fine too. It uses relative path linking, thank you for your review.

**Comment by hazerb** [comment]
i checked  usage of {{ url_for() }} function, but my code seems fine too. It uses relative path linking, thank you for your review.

File: practice-app/templates/coronavirus.html (hunk 2 above)
Comment (UkcaGreen) [reply practice-app/templates/coronavirus.html:108]: Then it can be stay that way.

**Pull request merged**